packages/core/src/transport.ts

```diff
@@ -1,6 +1,9 @@
 import { monitor } from './internalMonitoring'
 import { Context, deepMerge, DOM_EVENT, jsonStringify, noop, objectValues } from './utils'
 
+// https://en.wikipedia.org/wiki/UTF-8
+const HAS_MULTI_BYTES_CHARACTERS = /[^\u0000-\u007F]/g
```

Comment:
```suggestion
const HAS_MULTI_BYTES_CHARACTERS = /[^\u0000-\u007F]/
```
since we don't use match anymore